bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015A/To2015A.scala

```diff
@@ -91,8 +81,8 @@ object To2015A {
 
         // Add them back as children of this template parameters
         // data object ParamSet.
-        tp.addParamSet(targetMap(targetId).getParamSet(fact))
-        tp.addParamSet(condsMap(condsId).getParamSet(fact))
+        tp.addParamSet(PioXmlUtil.read(targetMap(targetId)).asInstanceOf[ParamSet])
```

Comment:
How do you know the cast is safe? Could `PioXmlUtil.read` return directly a `ParamSet`?